products/nextjs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Next.js
+category: framework
+tags: javascript-runtime
```

Comment:
```suggestion
tags: javascript-runtime vercel
```

We usually add a tag for the main contributor if it's a company and it is officially maintaining the project.